frameworks/hdfs/integration/tests/test_upgrade.py

```diff
@@ -24,10 +24,6 @@ def setup_module(module):
     uninstall()
 
 
-def teardown_module(module):
```

Comment:
Question answered offline, LGTM (answer was: ability to diagnose still-running service after failed test)